examples/deploy_patterns/pythonpath/README.md

```diff
@@ -0,0 +1,106 @@
+# Flyte SDK: Python Path Deployment Pattern
+
+This example demonstrates how to use `flyte.init` with the `root_dir` parameter when your Flyte workflows import modules from different directories. This is a common pattern when you have a project structure where workflow definitions are separated from your business logic modules.
```

Comment:
Can we mention that this is not the recommended approach if you're going to start a new repo?